frontend/public/components/pdb/PDBForm.tsx

```diff
@@ -0,0 +1,381 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import i18next from 'i18next';
+import {
+  ActionGroup,
+  Alert,
+  Button,
+  Form,
+  TextInput,
+  FormGroup,
+  GridItem,
+  Grid,
+  Text,
+  TextVariants,
+  Dropdown,
+  DropdownItem,
+  DropdownToggle,
+} from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import {
+  ButtonBar,
+  history,
+  resourcePathFromModel,
+  ExternalLink,
+  openshiftHelpBase,
+} from '@console/internal/components/utils';
+import { FieldLevelHelp } from '@console/internal/components/utils/field-level-help';
+import { PodDisruptionBudgetModel } from '../../models';
+import { k8sCreate, k8sPatch, PodDisruptionBudgetKind } from '../../module/k8s';
+import { SelectorInput } from '../utils';
+import { pdbToK8sResource, FormValues, intOrPercentage } from './pdb-models';
+import { MatchLabels } from '@console/dynamic-plugin-sdk/src/api/common-types';
+import { Patch } from '@console/internal/module/k8s';
+
+const AvailabilityRequirementPopover: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <FieldLevelHelp>
+      <Text component={TextVariants.h3}>{t('public~Availability requirement')}</Text>
+      <Text component={TextVariants.h5}>{t('public~maxUnavailable')}</Text>
+      <Text component={TextVariants.p}>
+        {t(
+          'public~An eviction is allowed if at most "maxUnavailable" pods selected by selector are unavailable after the eviction, i.e. even in absence of the evicted pod.',
+        )}
+      </Text>
+      <Text component={TextVariants.h5}>{t('public~minAvailable')}</Text>
+      <Text component={TextVariants.p}>
+        {t(
+          'public~An eviction is allowed if at least "minAvailable" pods selected by "selector" will still be available after the eviction.',
+        )}
+      </Text>
+      <Text component={TextVariants.h5}>{t('public~More information:')}</Text>
+
+      <ExternalLink
+        href={`${openshiftHelpBase}rest_api/policy_apis/poddisruptionbudget-policy-v1.html#poddisruptionbudget-policy-v1`}
+        text={t('public~PodDisruptionBudget documentation')}
+      />
+    </FieldLevelHelp>
+  );
+};
+
+// t('public~Requirement')
```

Comment:
I don't think we need the comment? I'd expect `i18next.t('Requirement')` below to be sufficient.